### DISCUSSION TITLE
Browser Driver issues in WSL

### DISCUSSION THREAD

**Discussion by agarwalshashwat** [seleniumbase/SeleniumBase#1723] [Q&A] [answered]
Hi ! I am running WSL instance and trying to do seleniumbase automation testing but the issue is pytest is unable to detect chromedriver or any driver when it is already there is the environment. How to deal with this ? Unable to find any solution online. Attaching the error below

![image](https://user-images.githubusercontent.com/66725004/216278099-6cece551-3ae5-4787-b90f-162a8e2d6655.png)

**Comment by mdmintz** [comment, marked as answer]
That error means that SeleniumBase can't find Chrome on your system. (Not the driver).

If Chrome isn't located at the usual location, you can change where SeleniumBase looks for it: https://github.com/seleniumbase/SeleniumBase/issues/1709
(for seleniumbase ``4.12.3`` or newer)

Example:

```bash
pytest test_demo_site.py --binary-location="/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
```

Also as an option for other [Syntax Formats](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/syntax_formats.md):

```python
from seleniumbase import Driver
from seleniumbase import js_utils
from seleniumbase import page_actions

driver = Driver(browser="chrome", binary_location="/Applications/Google Chrome.app/Contents/MacOS/Google Chrome")
try:
    driver.get("https://seleniumbase.io/apps/calculator")
    page_actions.wait_for_element(driver, "4", by="id").click()
    page_actions.wait_for_element(driver, "2", by="id").click()
    page_actions.wait_for_text(driver, "42", "output", by="id")
    js_utils.highlight_with_js(driver, "#output", loops=6)
finally:
    driver.quit()
```
